dbt/models/default/columns.md

```diff
@@ -0,0 +1,384 @@
+## permit_address_full
+
+{% docs column_permit_address_full %}
+Address where permitted work will take place.
+
+This field combines the "mailing address" field in the source data
+with the concatenated output of the address component fields like
+`address_street_dir`, `address_street_name`, etc. We prefer the
+mailing address if it exists, and otherwise we substitute the
+concatenated address components.
+
+This field usually includes the municipality name for municipalities
+outside Chicago. If the municipality name is missing, the address
+is either in Chicago, or we generated it based on the concatenation
+of the address component fields, which do not include municipality
+name.
+{% enddocs %}
+
+## permit_assessable
+
+{% docs column_permit_assessable %}
+Whether the permitted work will affect the property's assessed
+values.
+
+This field is set by CCAO permit specialists who review the permit's
+work description to determine assessability.
+
+Possible values for this variable are:
+
+- null: The permit has not been reviewed yet
+- `A`: Assessable
+- `B`: Non-Assessable
+{% enddocs %}
+
+## permit_filing_type
+
+{% docs column_permit_filing_type %}
+Filing type.
+
+Possible values for this variable are:
+
+- null
+- `PM`
+{% enddocs %}
+
+## permit_improvement_type
+
+{% docs column_permit_improvement_type %}
+Type of improvement indicated by the permit.
+
+Possible values for this variable are:
+
+- null: No improvement code assigned
+- `110 - NEW CONSTRUCTION - MAJOR IMPRVT`
+- `111 - NEW BUILDING`
+- `112 - ADDITIONS`
+- `113 - DORMERS`
+- `114 - OTHER - MAJOR NEW CONSTRUCTION`
+- `130 - NEW CONSTRUCTION - MINOR IMPRVMT`
+- `131 - SWIMING POOL - TENNIS COURT`
+- `132 - DRIVEWAYS - PATIOS - WOOD DECK`
+- `133 - FENCING (and Gates)`
+- `134 - GARAGE CARPORTS BARNS`
+- `135 - OTHER-MINOR NEW CONSTRUCTION`
+- `151 - BASEMENT ROOM - REC ROOM`
+- `152 - ATTIC ROOM - BATHROOMS`
+- `153 - FIREPLACE-CENTRAL AIR CONDITIONING`
+- `154 - OTHER - REMODELING`
+- `155 - ???`
```

Comment:
**praise:** Thanks for presumably copying or scraping these values from the HTML. What a pain.

What's the deal with the `???` values?

Reply:
Great question! 12 permits in the view have a `155` improvement code, and I can't determine a clear pattern based on their work descriptions. I've added this to my list of questions to discuss on Thursday.